packages/vue/src/components/ui-components/UserButton/UserButton.vue

```diff
@@ -1,13 +1,15 @@
 <script setup lang="ts">
 import { useClerk } from '../../../composables';
-import type { UserButtonProps } from '@clerk/types';
+import type { UserButtonProps, UserProfileProps } from '@clerk/types';
 import { ClerkHostRenderer, CustomPortalsRenderer } from '../../ClerkHostRenderer';
 import { computed, provide } from 'vue';
 import { UserButtonInjectionKey, UserProfileInjectionKey } from '../../../keys';
 import { useUserProfileCustomPages } from '../../../utils/useCustomPages';
 import { useUserButtonCustomMenuItems } from '../../../utils/useCustomMenuItems';
 
-type Props = Omit<UserButtonProps, 'customMenuItems'>;
```

Comment:
❓ Why did we not omit `customMenuItems` now?

Reply:
Updated!